nevergrad/functions/images/core.py

```diff
@@ -181,9 +181,13 @@ def make_folder_functions(
             )
         else:
             raise ValueError(f"{folder} is not a valid folder.")
+        index = 0
```

Comment:
please use itertools.islice instead, it's a better practice (one liner, less impact, more idiomatic)
That would just be `for data, target in itertools.islice(data_loader, 0, 100)` if I am correct

Reply:
Done, thx.